.github/workflows/doc-lint-test.yaml

```diff
@@ -246,6 +319,14 @@ jobs:
             echo "::error::Building html docs ${ret##*:}"
             exit 1
           fi
+          echo "BUILT_DOCS_PATH=$PWD/build/sphinx/html" >> $GITHUB_OUTPUT
+      - name: Upload HTML Docs
+        uses: actions/upload-artifact@v4
+        with:
+          name: HTML Docs
+          path: ${{ steps.build-html-docs.outputs.BUILT_DOCS_PATH }}
+          if-no-files-found: error
+          overwrite: true
```

Comment:
Is `overwrite: true` needed here? If so, why?

Reply:
See above